app/controllers/listings_controller.rb

```diff
@@ -488,7 +488,8 @@ def update
 
   def finalise_update(listing, shape, community, update_successful, old_availability)
     if update_successful
-      listing.location.update_attributes(params[:location]) if listing.location
+      location_params = permit_origin_loc_params(params.require(:listing))
```

Comment:
Again, this information would be cool to have in the pr description/comment description.